library/item/item_group/src/main/java/org/quiltmc/qsl/itemgroup/impl/QuiltCreativePlayerInventoryScreenWidgets.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2016, 2017, 2018, 2019 FabricMC
+ * Copyright 2021 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.itemgroup.impl;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.function.Consumer;
+
+import com.mojang.blaze3d.systems.RenderSystem;
+import org.jetbrains.annotations.ApiStatus;
+
+import net.minecraft.client.gui.screen.ingame.CreativeInventoryScreen;
+import net.minecraft.client.gui.widget.ButtonWidget;
+import net.minecraft.client.util.math.MatrixStack;
+import net.minecraft.item.ItemGroup;
+import net.minecraft.text.LiteralText;
+import net.minecraft.text.Text;
+import net.minecraft.text.TranslatableText;
+import net.minecraft.util.Identifier;
+
+@ApiStatus.Internal
+public final class QuiltCreativePlayerInventoryScreenWidgets {
+	private static final Identifier BUTTON_TEXTURE = new Identifier("qsl_items_item_group", "textures/gui/creative_buttons.png");
```

Comment:
```suggestion
	private static final Identifier BUTTON_TEXTURE = new Identifier("qsl_item_group", "textures/gui/creative_buttons.png");
```

Namespace containing two times "item" feels a bit weird, namespace don't need to include the library's name.